### ISSUE TITLE
401 Unauthorized for Managed User

### ISSUE THREAD

**Issue by camsterguy** [I-am-PUID-0/pd_zurg#82]
**Describe the bug**

pd_zurg is stating 401 unauthorized for user i added, please check token 

**To Reproduce**

Adding a managed local user via Plex home, using firefox dev tools to identify the token for the user, adding the user to pd_zurg settings with the token

**Expected behavior**

Using the other user to pd_zurg so my other user can add stuff to their watchlist which will also download

**Screenshots**

Full error:
Nov 18, 2024 03:14:18 - INFO - plex_debrid subprocess: [18/11/24 03:14:18] plex error: (401 unauthorized): token for user 'sammie' does not seem to work. check your plex user settings.

**Please complete the following information:**
 - OS: Linux
 - Version: Mint
 - Supporting Applications: Docker Containers

**Additional context**

Does zurg not support managed users? Maybe I would just need to delegate access to another plex email / user as opposed to plex home managed users

Thanks for any help!

**Comment by camsterguy** [comment]
When adding the user, what should i use for the username? The exact name I made for the managed user?

This is an API readout, and this is the user I added. I see this token correctly (accessToken)

Should I be using one of these values for the username?

```
<MediaContainer friendlyName="myPlex" identifier="com.plexapp.plugins.myplex" machineIdentifier="REDACTED" size="1">
<SharedServer id="REDACTED" username="" email="" userID="REDACTED" accessToken="REDACTED" name="REDACTED" acceptedAt="1731898511" invitedAt="1731898511" allowSync="1" allowCameraUpload="0" allowChannels="0" allowTuners="1" allowSubtitleAdmin="0" owned="1" allLibraries="1" filterAll="" filterMovies="" filterMusic="" filterPhotos="" filterTelevision="">
<Section id="130523387" key="2" title="Movies" type="movie" shared="1"/>
<Section id="130523408" key="3" title="TV Shows" type="show" shared="1"/>
<Section id="130615340" key="4" title="TV Shows" type="show" shared="1"/>
</SharedServer>
</MediaContainer>
```

**Comment by I-am-PUID-0** [comment]
The primary plex admin should be set with the pd_zurg envs, then I believe the users would be added through the plex_debrid UI: https://github.com/itsToggle/plex_debrid/issues/501

**Comment by camsterguy** [comment]
> The primary plex admin should be set with the pd_zurg envs, then I believe the users would be added through the plex_debrid UI: [itsToggle/plex_debrid#501](https://github.com/itsToggle/plex_debrid/issues/501)

Thanks for the reply!
The admin is working fine.

i created a managed user, named “Sammie”, from the plex web interface

I then launch the plex debrid ui using “docker attach pd_zurg”, go to content settings, add the user “Sammie” and the token of that user that I found by using Firefox’s network tools while logged into the managed user, clicking on a show and inspecting the network headers to copy the “X-PlexToken”. And I still get the 401 error.

I set up my environment using this tutorial:
https://docs.google.com/document/d/1wGzoV11vG2jiSiKm_EoShrcig70pWYPDYiYbAXT5jRQ/mobilebasic#heading=h.nu4h35dxqc4q
